api/v1alpha1/addonimageset_types.go

```diff
@@ -53,6 +53,14 @@ type AddonImageSetSpec struct {
 	// +optional
 	// Configs to be passed to the subscription OLM object.
 	SubscriptionConfig *mtsrev1.SubscriptionConfig `json:"subscriptionConfig"`
```

Comment:
Shouldn't this be `config` instead of `subscriptionConfig`?
Ref: https://github.com/Ankit152/managed-tenants-cli/blob/main/managedtenants/schemas/imageset.schema.yaml#L28-L30

Reply:
Yes

Reply:
Okay. Updating json fields.